File: Assets/Scripts/OSC/FFTScale.js
public var channel : int = 1;
public var fftObjects : GameObject[];
private var gain : float = .5;

function OSCfft (input) {
	if(input.channel == channel) {
		for(i=0; i<24; i++) {
			fftObjects[i].transform.localScale.y = input.ranges[i] *gain;
		}
	}
}

File: Assets/Scripts/OSC/FFTScale.cs
using UnityEngine;
using System.Collections;

public class FFTScale : MonoBehaviour 
{
	public int channel = 1;
	public GameObject[] fftObjects;
	private float gain = 0.25f;
	
	void OSCfft (OSCHandler.FftData fftData) {
		if(fftData.channel == channel) {
			for(int i=0; i<24; i++) {
				//Debug.Log("fft");
				//Vector3 objectLocalScale = fftObjects[i].transform.localScale;
				//objectLocalScale.y = fftData.ranges[i] *gain;
				Vector3 offset = new Vector3(1, fftData.ranges[i] *gain, 1);
				fftObjects[i].transform.localScale = offset;
			}
		}
	}
}
